src/requirements/data/majors/ce.ts

```diff
@@ -0,0 +1,126 @@
+import { CollegeOrMajorRequirement } from '../../types';
+import { includesWithSingleRequirement, includesWithSubRequirements } from '../checkers-common';
+
+const civilRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Engineering Distribution',
```

Comment:
this requirement overlaps with the engineering distribution requirement, so it's better to mark it as allowDoubleCounting. You might need to merge in latest master